app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
```

Comment:
shouldn't it be a numeric type?

Reply:
No, cause every data, which comes from html attributes is string. Then I should parse it, if it neccesery.